src/Error/JsonApiExceptionRenderer.php

```diff
@@ -97,6 +97,10 @@ public function validation($exception)
         // set data and send response
         $this->controller->response->type('jsonapi');
         $this->controller->response->body($json);
+        $this->controller->response->cors($this->controller->request)
```

Comment:
Sorry yesterday i have problem with my mind.... :smile: i used plugin cors to do it, but when have error CORS no works correctly.